shelf/search/sorter.py

```diff
@@ -29,7 +29,7 @@ def standard_sort(result):
         # `None` it returns "0" as this creates the proper sort order. Secondly, it uses
         # distutils.version.LooseVersion to facilitate the version sort.
         def version_sort(result):
-            value = standard_sort(result)
+            value = str(standard_sort(result))
```

Comment:
Oh ya know what? Thats probably due to that if statement never being true anymore